### PR TITLE
Updated explainer to reflect new API surface

### PR DESCRIPTION
**Pull request by rstz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,6 +119,19 @@ interface NativeIOManager {
...
                                          DOMString new_name);
+  
+  // Requests new capacity (in bytes) for usage by the current execution
+  // context.
+  [ CallWith = ScriptState, RaisesException ] Promise<unsigned long long>
+  requestCapacity(unsigned long long requested_capacity);
+  
+  // Releases unused capacity (in bytes) from the current execution context.
+  [ CallWith = ScriptState, RaisesException ] Promise<unsigned long long>
+  requestCapacity(unsigned long long requested_capacity);
```
Comment (fivedots) [inline README.md:130]: I think you meant release here

Comment (fivedots) [reply README.md:130]: Could you also expand the comment to mention what the return value is, both here and in requestCapacity?

File: README.md [hunk 2]
```diff
@@ -98,13 +98,14 @@ The following are our draft proposals for the interface, written in Web IDL.
...
-All filesystem calls are accessible through a global NativeIO instance.
+All filesystem calls are accessible through a global Storage Foundation
+instance.
 
 
 ```webidl
 // IMPORTANT: filenames are restricted to lower-case alphanumeric and
 // underscore (a-z, 0-9, _).
-interface NativeIOManager {
+interface NativeIOFileManager {
```
Comment (tomayac) [inline README.md:108]:
```suggestion
interface StorageFoundationFileManager {
```

Can this be renamed, too?

File: README.md [hunk 3]
```diff
@@ -249,18 +266,36 @@ prevent concurrent, unsafe writes to the same file. Our current approach is
...
+The web application must explicitly ask for capacity before storing any new
+data. This request will be granted according to the browser's quota
+guidelines. The application can then use the capacity to execute write
+operations. The application can also delete or overwrite data to make space for
+new data without issuing a new request.
+
+This means that anytime an application starts a new Javascript execution context
+(e.g., new tab, new worker, reloading the page), it must make sure it owns
+sufficient capacity before writing new data. It must therefore either
+delete/shrink old files or request additional capacity.
```
Comment (fivedots) [inline README.md:283]: Consider explicitly mentioning overwritring old files as well e.g. not delete or shrink an existing file, but always write within the existing boundaries.

**Review by tomayac — COMMENTED** [review]
Just left a comment about a remaining instance of the "Native I/O" name.

File: README.md [hunk 4]
```diff
@@ -249,18 +266,36 @@ prevent concurrent, unsafe writes to the same file. Our current approach is
...
+
 ## Trying It Out
 
 A prototype of Storage Foundation API is available in Chrome Canary. To enable
-it, launch Chrome with the `-enable-blink-features=NativeIO `flag or enable
-“Experimental Web Platform feature” in ["chrome://flags"](chrome://flags).
+it, launch Chrome with the `-enable-blink-features=StorageFoundation `flag or
+enable “Experimental Web Platform feature” in
+["chrome://flags"](chrome://flags).
```
Comment (fivedots) [inline README.md:290]: Could you also add a sentence pointing people to our Origin Trial registration page?

i.e. https://developer.chrome.com/origintrials/#/view_trial/1673087261568139265

**Review by fivedots — APPROVED** [review]
LGTM, thanks for taking care of this!

File: README.md (hunk 2 above)
Comment (rstz) [reply README.md:108]: Good catch, I'll rename this soon - first in Chromium and, as soon as that's landed, in the explainer. Unfortunately,  `NativeIOFileManager` is still visible in the current version (but only if you look at it very closely in DevTools).

File: README.md (hunk 1 above)
Comment (rstz) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (rstz) [reply README.md]: Done

File: README.md (hunk 4 above)
Comment (rstz) [reply README.md:291]: Done.

**Pull request merged**